conf/nginx.conf

```
@@ -1,7 +1,9 @@
 #sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
-location __PATH__/ {
+location ^__PATH__/ {
```

```suggestion
location __PATH__/ {
```

Not sure that we need `^`, `^` is for regex mode which is when we have `location ~` I think ?